### PR TITLE
Use a get data function

### PR DESCRIPTION
**Pull request by jeromedockes**
`get_data` is deprecated in nibabel

### CODE REVIEW

File: nistats/reporting/_get_clusters_table.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 import nibabel as nib
 from scipy import ndimage
 from nilearn.image.resampling import coord_transform
+from nistats.utils import get_data
```
Comment (bthirion) [inline nistats/reporting/_get_clusters_table.py]: I'd rather use it from nilearn directly rather than copying to nistats.

**Comment by jeromedockes** [comment]
> I'd rather use it from nilearn directly rather than copying to nistats.

but that would fail for users who don't have a recent version of nilearn

**Comment by kchawla-pi** [comment]
>> I'd rather use it from nilearn directly rather than copying to nistats.

> but that would fail for users who don't have a recent version of nilearn

Jerome is right. If we are to stop Nistats development and keep it for users in as complete state as possible it is best to incorporate this whole feature. 
Other wise, we might as well not bother since Nistats won't be long once we release it as nilearn.stats

**Comment by bthirion** [comment]
Merging --- to be on par with Nilearn.
Thx !

**Pull request merged**